spec/models/user_spec.rb

```diff
@@ -18,11 +21,30 @@
       expect(user.video_in_queue?(video)).to be(true)
     end
 
-    it 'returns fals if the video is not in users video queue' do
+    it 'returns false if the video is not in users video queue' do
       another_video = Fabricate(:video)
       expect(user.video_in_queue?(another_video)).to be(false)
     end
   end
 
+  describe '#follow' do
+    it 'creates a relationship to another user' do
+      user.follow another_user
+      expect(user.followed_users).to include(another_user)
+      expect(another_user.followers).to include(user)
+    end
+  end
 
+  describe '#unfollow' do
+
+    before(:each) do
+      user.follow another_user
+    end
+
+    it 'removes the relationship' do
+      expect(user.followed_users).to include(another_user)
```

Comment:
I think this step is not necessary - you already test follow in previous method, so asserting this in removing the relationship may be redundant.